config/config-api/test/com/thoughtworks/go/config/MergeCruiseConfigTest.java

```diff
@@ -299,56 +298,22 @@ public void addPipelineWithoutValidation_shouldAddPipelineToMain()
         assertThat(cruiseConfig.hasPipelineNamed(new CaseInsensitiveString("pipe3")), is(true));
 
     }
+
     @Test
-    public void addPipelineWithoutValidation_shouldFailToAddPipelineWhenItExistsInPartialConfig()
```

Comment:
What does it mean that these tests are removed now? I don't see replacements for all. Does it mean it was wrong, or not needed or that functionality is gone?